src/scripts/main.js

```diff
@@ -0,0 +1,120 @@
+const formContainer = document.querySelector('.form__container');
+const addButton = document.querySelector('.form__button--add');
+const submitButton = document.querySelector('.form__button--submit');
+const MAX_INPUT_COUNT = 6;
+
+const regEx = /.+?(?=[?]passwords=)/;
+
+let activeInputs = 0;
+
+const checkInputs = inputs => {
```

Comment:
nadałeś nazwę `checkInputs`,
funkcja ta zwraca wartość logiczną true lub false -> więc nazwa powinna na to wskazywać:
`isInputsChecked` / `isInputsCorrect` / `isFormValid` etc.
przy okazji również walidujesz form i jednocześnie wstrzykujesz errorMessage. czyli nie dość, że nazwa jest myląca to w dodatku funkcja robi side-effecty.

możesz wydzielić osobny plik, w którym będziesz trzymał tylko funkcje na potrzeby walidacji np: `validation.js`